circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/aws/RetryableTransferManagerFactory.java

```diff
@@ -24,15 +24,16 @@
 import com.hotels.bdp.circustrain.s3s3copier.S3S3CopierOptions;
 
 @Component
-public class TransferManagerFactory {
+public class RetryableTransferManagerFactory {
```

Comment:
Why is this change necessary ? A transferManagerFactory is a Factory which returns different types of transferManagers, in your case now, you will return a RetryableTransferManager.

Please let me know if I am missing anything.

Reply:
Hey, can I resolve this conversation? Thanks!

Reply:
Yes Please.